### PR TITLE
Add excludeBodyId option to JSON reader

### PR DESCRIPTION
**Pull request by superfashi**
This PR adds support of `excludeBodyId` attributes to `Env::LoadJSON`, which indicates what body ids are excluded from loading into the environment.

### CODE REVIEW

**Comment by rdiankov** [comment]
changelog and patch update please

File: src/libopenrave-core/jsonparser/jsonreader.cpp [hunk 1]
```diff
@@ -310,6 +313,10 @@ class JSONReader
                     throw OPENRAVE_EXCEPTION_FORMAT("Has body with no name in file '%s', so cannot load the scene: %s", _filename%orjson::DumpJson(rTempKinBodyInfo), ORE_InvalidArguments);
                 }
             }
+            if ( _excludeBodyIds.count(pKinBodyInfo->_id) ) {
+                itBodyInfo = envInfo._vBodyInfos.erase(itBodyInfo);
```
Comment (ziyan) [inline src/libopenrave-core/jsonparser/jsonreader.cpp:317]: @superfashi does this work? what if the bodyinfo is used later by a reference?

Comment (superfashi) [reply src/libopenrave-core/jsonparser/jsonreader.cpp:317]: Looks to me that `_ProcessEnvInfoBodies` which uses `_ExpandRapidJSON` already resolves the `referenceUri`

**Comment by superfashi** [comment]
> changelog and patch update please

Sorry about that. Updated.

File: src/libopenrave-core/jsonparser/jsonreader.cpp (hunk 1 above)
Comment (rdiankov) [reply src/libopenrave-core/jsonparser/jsonreader.cpp:317]: use `std::find` instead of `count`. thanks

Comment (superfashi) [reply src/libopenrave-core/jsonparser/jsonreader.cpp:317]: Can we use `set::find`?

Comment (cielavenir) [reply src/libopenrave-core/jsonparser/jsonreader.cpp:317]: .find is better; std::find is O(N) for (unordered_)set

Comment (cielavenir) [reply src/libopenrave-core/jsonparser/jsonreader.cpp:317]: By the way I'm personally surprised about this comment as I thought .count was (kind of) Mujin standard, eg https://github.com/rdiankov/openrave/blob/production/src/libopenrave/kinbodyjoint.cpp#L2417

Comment (rdiankov) [reply src/libopenrave-core/jsonparser/jsonreader.cpp:317]: Well... i'm personally surprised that code made it into openrave

**Comment by rdiankov** [comment]
thanks

**Pull request merged**